### PR TITLE
686-config-list-add-deleted-tagstyle

### LINKED ISSUE #686: Config List - Add deleted tag/style
- Change the row text color to gray if deleted_at != null
- Add light red deleted badge after the name

### PR DESCRIPTION
**Pull request by mainawycliffe**
Closes #686

### CODE REVIEW

File: src/components/ConfigList/Cells/ConfigListNameCell.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import { ConfigItem } from "../../../api/services/configs";
+import { Badge } from "../../Badge";
+
+function ConfigListNameCell({ row, getValue }: CellContext<ConfigItem, any>) {
+  const isDeleted = !!row.original.deleted_at;
+
+  return (
+    <div className="flex flex-row space-x-2 items-center">
+      <span className="truncate">{getValue()}</span>
+      {isDeleted && <Badge text="deleted" colorClass="text-white bg-red-500" />}
```
Comment (moshloop) [inline src/components/ConfigList/Cells/ConfigListNameCell.tsx:12]:
```suggestion
      {isDeleted && <Badge text="deleted" colorClass="text-white bg-gray-400" />}
```

**Comment by moshloop** [comment]
@mainawycliffe  can we add:

- [ ] Filter to hide/shot deleted items
- [ ] The gray should not apply to the summary row (unless all items are deleted)
- [ ] Sorting should be split into 2 - first sort non-deleted items, and then sort deleted items after current
- [ ] Use the deleted date in the last_updated column with a gray trash icon
- [ ] Add all 3 dates (if they exist) into the tags/detail panel on the detail page

**Comment by mainawycliffe** [comment]
@moshloop Yeah, I will add those in.

**Comment by mainawycliffe** [comment]
@moshloop can you please confirm whether this is the kind of sorting you had in mind

**Comment by moshloop** [comment]
@mainawycliffe It looks like the sorting works with no grouping, but with grouping enabled the deleted items come first

**Pull request merged**